lib/util/concerns/hud_lists_2024.rb

```diff
@@ -1779,11 +1784,12 @@ def project_completion_status(id, reverse = false)
     # R17.A
     def early_exit_reasons
       {
-        1 => 'Left for other opportunities - independent living',
-        2 => 'Left for other opportunities - education',
-        3 => 'Left for other opportunities - military',
-        4 => 'Left for other opportunities - other',
-        5 => 'Needs could not be met by project',
+        1 => 'Criminal activity/destruction of property/violence',
+        2 => 'Non-compliance with project rules',
+        3 => 'Non-payment of rent/occupancy charge',
+        4 => 'Reached maximum time allowed by project',
+        5 => 'Project terminated',
+        6 => 'Unknown/disappeared',
```

Comment:
Oooh, this looks like we just had the wrong set.  And this is why we wanted to automate it.